src/ngx-gallery-scroll-overview/ngx-gallery-scroll-overview.component.html

```diff
@@ -0,0 +1,8 @@
+<div id="previewContainer" class="drag-boundary image-box"
```

Comment:
Use kebab-case also for ids (`preview-container`)
You won't need css classes here if you use the id instead of the class in the scss file (you don't use drag-boundary anyway)